template/python/index.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import FastAPI, Request, Response
+from function.handler import handler
+from fastapi.responses import JSONResponse
+import os
+
+app = FastAPI()
+
+@app.route('/', methods=['GET', 'PUT', 'POST', 'PATCH', 'DELETE'])
+@app.route('/{path:path}', methods=['GET', 'PUT', 'POST', 'PATCH', 'DELETE'])
+def call_handler(request: Request):
+    return JSONResponse(dict(handler(request)), status_code=200)
+
+port = int(os.environ.get("PORT", 3000))
+
+if __name__ == "__main__":
+    import uvicorn
+    uvicorn.run("index:app", host="0.0.0.0", port=port, log_level="info", reload=True)
```

Comment:
Is it possible to do the same thing than the `node-19` template ? Because here, you send always a `200 Ok` to the caller. The idea behind those runtimes at the moment is to give the ability to the user to control the response. If he want to return `400 Bad Request`, he must be able to do it.

Reply:
fixed